### ISSUE TITLE
"Connect to AWS" does nothing

### ISSUE THREAD

**Issue by LeonardoSanBenitez** [aws/aws-toolkit-vscode#2906]
## Problem

After installing the plugin, I click in "AWS: Connect to AWS" in the command palette but nothing happens. Absolutely nothing. 
I have the credential files (`~/.aws/config` and `~/.aws/credentials`) already configured, and I can use the AWS CLI normally.

If I delete the credential files and try to run "Create Credentials Profile" in the command palette, nothing happens as well.

Also, when I access the plugin from the Activity Bar, it just loads forever.
![image](https://user-images.githubusercontent.com/31520774/191799112-8cb4986a-37a9-4892-bc96-86546f87fbb1.png)

## Steps to reproduce the issue

1. Ctr+shit+P to open the command palette
2. Click on AWS: Connect to AWS

Expected behavior: I would be able to select the profile.
Actual behavior: nothing happens

## System details (run the `AWS: About Toolkit` command)

-   OS: ubuntu 20.04
-   Visual Studio Code version: 1.71.2-1663191218, installed from apt
-   AWS Toolkit version: 1.50.0

**Comment by justinmk3** [comment]
This is very unusual, it seems like the extension code is not executing. Can you provide the logs (after removing any private info)? 

1. In VSCode settings, ensure that "aws log level" is "debug".
2. Restart VSCode and try the steps to reproduce the issue.
3. Use the `AWS: View Toolkit Logs` command to get the logs. 
    - If that _also_ fails, see if there's a "AWS Toolkit Logs" Output channel:
      <img width="406" alt="image" src="https://user-images.githubusercontent.com/55561878/191843454-cc2d531d-a23c-430a-8a80-018cc1891fb3.png">

**Comment by LeonardoSanBenitez** [comment]
It looks like it was a problem from vscode itself, but I ended up solving it before exporting the logs.
I first tried reinstalling vcode, but it didn't solved. At the same time, I noted a problem with a subprocess of vscode executing `rg` and consuming a lot of CPU, and in the [issue 1257](https://github.com/microsoft/vscode-remote-release/issues/1257) they suggest that it could be because of large folders.

I removed the open folders, and suddenly aws-toolkit-vscode worked normally.
